framework/lib/components/toolbox/toolbox.tsx

```diff
@@ -60,7 +60,7 @@ export const Toolbox = ({
   resizeLimit = 'full',
   zIndex = coreZIndex.overlay,
   ...rest
-}: Omit<ToolboxProps, 'zIndex'> & { zIndex: number }) => {
+}: Omit<ToolboxProps, 'zIndex'> & { zIndex?: number }) => {
```

Comment:
can't we do this `Omit` operation in `ToolboxProps`?
Doing this here increases the chance of devs being confused by ToolboxProps type declarations.